src/libraries/System.IO.FileSystem/tests/Base/BaseGetSetTimes.cs

```diff
@@ -16,7 +16,10 @@ public abstract class BaseGetSetTimes<T> : FileSystemTest
         private static string driveFormat = PlatformDetection.IsInAppContainer ? string.Empty : new DriveInfo(Path.GetTempPath()).DriveFormat;
 
         protected static bool isHFS => driveFormat != null && driveFormat.Equals(HFS, StringComparison.InvariantCultureIgnoreCase);
-        protected static bool isNotHFS => !isHFS;
+        protected static bool isBrowser => PlatformDetection.IsBrowser;
+
+        protected static bool isHFSOrBrowser => isBrowser || isHFS;
```

Comment:
Generally, it's better to describe the intention, not the code logic. In this case you could name it `HasMillisecondsResolution` instead or something like that